.travis/build-and-test.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
+
+set -e
+
+source .travis/common.sh
+
+##########################################################################
+
+echo
```

Comment:
Can you create a new issue to replace these with calls to replace these echos with things like https://github.com/litex-hub/litex-conda-packages/blob/9b201004a699af707afc0745e57a06c223e4f545/.travis/common.sh#L32-L43 ?